openapi.json

```diff
@@ -0,0 +1,1413 @@
+                          "display_text"
+                        ]
+                      },
+                      "type": "array"
+                    },
+                    "software": {
+                      "properties": {
+                        "id": {
+                          "type": "integer"
+                        },
+                        "name": {
+                          "type": "string"
+                        },
+                        "version": {
+                          "type": "string"
+                        },
+                        "bundle_identifier": {
+                          "type": "string"
+                        },
+                        "source": {
+                          "type": "string"
+                        },
+                        "release": {
+                          "type": "string"
+                        },
+                        "vendor": {
+                          "type": "string"
+                        },
+                        "arch": {
+                          "type": "string"
+                        },
+                        "generated_cpe": {
+                          "type": "string"
+                        },
+                        "vulnerabilities": {
+                          "items": {
+                            "properties": {
+                              "cve": {
+                                "type": "string"
+                              },
+                              "details_link": {
+                                "type": "string"
+                              }
+                            },
+                            "additionalProperties": false,
+                            "type": "object",
+                            "required": [
+                              "cve",
+                              "details_link"
+                            ]
+                          },
+                          "type": "array"
+                        },
+                        "hosts_count": {
+                          "type": "integer"
+                        }
+                      },
+                      "additionalProperties": false,
+                      "type": "object",
+                      "required": [
+                        "id",
+                        "name",
+                        "version",
+                        "source",
+                        "generated_cpe",
+                        "vulnerabilities"
+                      ]
+                    },
+                    "error": true
+                  },
+                  "additionalProperties": false,
+                  "type": "object",
+                  "required": [
+                    "hosts"
+                  ]
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/fleet/hosts/delete": {
+      "post": {
+        "operationId": "post_hosts_delete",
+        "requestBody": {
```

Comment:
`requestBody` would need quite a bit of work - we use structs that mix url, query, json in the request body and even custom fields (like `list_options` that are ultimately in the query). Moreover, some endpoints sometimes support only a subset of what the struct supports.